Makefile

```diff
@@ -7,6 +7,8 @@ NOTIFY_CREDENTIALS ?= ~/.notify-credentials
 
 CF_APP = document-download-api
 
+CF = $(if $(shell which cf-cli-6.43), cf-cli-6.43, cf)
```

Comment:
after we roll out this app, my hope is that we'll be able to make sure cf 6.43 works for old apps too, and then not have both cfs in parallel - we don't need all apps to be zdt for that to happen, this was just a precaution.